cmake/ctest/drivers/atdm/cts1empire/drivers/Trilinos-atdm-cts1-intel-19.0.5_openmpi-4.0.1_openmp_static_dbg.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+#export SLURM_CTEST_TIMEOUT=1:00:00
+if [ "${Trilinos_TRACK}" == "" ] ; then
+  export Trilinos_TRACK=ATDM
+fi
+$WORKSPACE/Trilinos/cmake/ctest/drivers/atdm/cts1/local-driver.sh
```

Comment:
Might be good to delete support for intel-19 in this new 'cts1empire' env as EMPIRE can't use intel-19.